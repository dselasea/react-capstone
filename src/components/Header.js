import React from 'react';

const Header = () => (
  <header style={{ color: '#fff' }}>
    <h1 className="header-title">Crypto Currency Application</h1>
  </header>
);

export default Header;
