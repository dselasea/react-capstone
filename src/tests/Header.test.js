import renderer from 'react-test-renderer';
import Header from '../components/Header';

describe('Home', () => {
  it('renders correctly', () => {
    const headerComponent = renderer
      .create(<Header />)
      .toJSON();
    expect(headerComponent).toMatchSnapshot();
  });
});
