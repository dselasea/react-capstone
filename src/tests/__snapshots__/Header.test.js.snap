// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<header
  style={
    Object {
      "color": "#fff",
    }
  }
>
  <h1
    className="header-title"
  >
    Crypto Currency Application
  </h1>
</header>
`;
